lstchain/scripts/lstchain_dvr_pixselector.py

```diff
@@ -496,6 +506,33 @@ def main():
                 time.sleep(300)
```

Comment:
what was the reason for this `sleep` here?

Reply:
I introduced this because at some point we had many errors in writing out the pixel masks, related to glitches in the system @ LP. So I introduced this: in case of error it waits for five minutes and tries again to write out.  It does it a maximum number of times.